content/docs/ref/runner.md

```diff
@@ -0,0 +1,5 @@
+# Command Reference: `runner`
+
+Starts a runner (either via any supported cloud compute provider or locally
+on-premise -- see
+[cloud compute resource credentials](/doc/self-hosted-runners#cloud-compute-resource-credentials)).
```

Comment:
Minor: perhaps a more general link to the top of `/doc/self-hosted-runners` instead?